lang/java/BUILD.md

```diff
@@ -44,6 +46,12 @@ To perform "fast" build, which skips tests and all code quality enforcement tool
 
     mvn clean install -DskipTests -TC1 -P!code-quality
 
+
+  * The REEF unit tests require a number of open files which is greater than the default open file limit on a number of Linux distributions such as Ubuntu 16.04/16.10.  This limit is controlled in the shell by the "ulimit -n" command
+
+  * The builds for REEF have been tested on Ubuntu v16 , Win10, Win2012, Win2016,OSX
```

Comment:
Maybe we can move this to the `Prerequisites` section? E.g.

````
Prerequisites
-------------

  * One of the supported OSs:
    * Windows: We have had success with Windows 8, 10, Server 2012 and Server 2016.
    * Linux: We have had success with Ubuntu 16.04 and 16.10.
    * macOS: ??
```

@bgchun Can you help us out here with the version of macOS you are running?